### DISCUSSION TITLE
NB 15 Linux installer?

### DISCUSSION THREAD

**Discussion by Jelle-S** [apache/netbeans#4637] [General]
For NetBeans 14 and earlier, there was a Linux `.sh` installer available for download (see https://netbeans.apache.org/download/nb14/), but for NetBeans 15 there doesn't seem to be one (https://netbeans.apache.org/download/nb15/)?

**Comment by geertjanw** [comment]
Linux users: use the binary zip, Snap, or our community installers (there is a .deb without JDK too), https://codelerity.com/netbeans/. This has been discussed on dev@ a little while back. Linux installer is a little broken and no-one has fixed it yet.
